aiida/tools/visualization/graph.py

```diff
@@ -187,6 +190,19 @@ def pstate_node_styles(node):
     return node_style
 
 
+override_styles_dict = {    # pylint: disable=invalid-name
```

Comment:
The reason `pylint` complains here is that [PEP8 suggests](https://www.python.org/dev/peps/pep-0008/#constants) constants to be named in all caps (`OVERRIDE_STYLES_DICT`).

Since we're generally following that guideline (see e.g. [hashing.py](https://github.com/aiidateam/aiida-core/blob/develop/aiida/common/hashing.py)), I'd suggest renaming the variable and dropping the `pylint: disable` statement here.

Reply:
Thanks! Here I use lowercase to makes it look similar to other functions that do style collecting. Again, it seems unnecessary. :crying_cat_face: 
Changed.

Reply:
No worries, just nitpicking here 🙂

On second thought, do we want this constant to be used outside this module, or should it remain an implementation detail here? If it's the latter, we should probably add a leading underscore, and remove it from the ``__all__``.

Reply:
I guess it might be used outside this module just as other style setting functions.

Reply:
Ok, I'll let @ramirezfranciscof decide that one.

Reply:
So, I don't think this would be used in any other part of the aiida-core, if that is what you meant. If you intended this to be exposed to users so that they can "pick" the style from this dict and pass that to the `recurse_whatever` functions when manually using them, I guess it is ok to expose it, but I personally doubt that this `dict` is going to get much bigger or that people are going to use it to customize their plots. Personally I would err on the side of caution and not expose it unless we really think it will be used this way, but I will leave this up to you @unkcpz .

Reply:
I did it in first place since I expose and use it in `verdi_graph`. I agree that there is no chance that others would reuse this dict. 
Erased it from __all__ and add a leading underscore.